### ISSUE TITLE
User Interface is "Clipping"

### ISSUE THREAD

**Issue by S3-Collaborater** [pulse0ne/eq8#35]
"The EQ's interface is clipping as if it's been cropped. I can only interact with half of three knobs right now."
-I've tried rebooting chrome and my PC.
-Deleted and Re-installed the extension.
-Hopefully this bug will fix itself with an update
-Or maybe somebody knows what's cracking?

Thanks for any tips or tricks,
-S3iun
eq8 user since '19 ( best eq extension there is. )

![Screenshot 2023-08-20 190932](https://github.com/pulse0ne/eq8/assets/142721495/8a370a8a-0c24-4406-ba30-3a0670da6318)

**Comment by pulse0ne** [comment]
Yeah, it looks like Google changed something that breaks the responsive sizing. I’m going to work on a fix today. I’m also working on a replacement plugin that is compliant with manifest v3. Eventually, Google is going to disable all v2 plugins, and the effort to make eq8 compliant is too great (basically a re-write, so I started fresh). However, the new plugin will have many new features, like customizable themes, import/export of presets, an unlimited number of EQ bands, and will work on all sites (like bandcamp, which eq8 doesn’t work on). I will update the issue here when eqplus is released so you know.

**Comment by pulse0ne** [comment]
I’m also working on writing a system-wide Windows equalizer. It will work similarly to Equalizer APO (but with a much better user interface)

**Comment by pulse0ne** [comment]
I have submitted a fix for review by Google. I will update the issue here when it has been accepted.

**Comment by S3-Collaborater** [comment]
Wow, amazing work! Sounds like a big project to redesign everything for v3, when it's already the best eq plugin on v2.
But updates keep coming. Thank you kindly sir for the fine work you do. If there's anything we can do to support this effort, please let us know! Do you have a Patreon?
Looking forward to any progress!
Please keep us posted on anything forthcoming!

**Comment by pulse0ne** [comment]
It looks like the new version was accepted by Google. It should be updated next time chrome starts up!

**Comment by pulse0ne** [comment]
@S3-Collaborater I have submitted version 1.0.0 of `eq+` to the chrome web store. I will update here when it gets accepted and can be used. It works much better than eq8. https://github.com/pulse0ne/eqplus

**Comment by S3-Collaborater** [comment]
Nice job! I like the customization!
One or two bugs I found.
If I want to type in a value, I cannot input a negative value or backspace to 0 if it is not zero.
Also, when readjusting nodes, my values slip If I cannot click them quickly enough.

**Comment by pulse0ne** [comment]
> Nice job! I like the customization!
> One or two bugs I found.
> If I want to type in a value, I cannot input a negative value or backspace to 0 if it is not zero.
> Also, when readjusting nodes, my values slip If I cannot click them quickly enough.

Yeah, I’ve got a few things that didn’t transfer smoothly from Vue to React, so I will get to work on those. I think the slippage will go away when I change the frequency response plot line to be per-node. Right now there’s a bit of code that tries to get the node to stick to the line, but that isn’t accounted for in the click handler logic.

thanks for the feedback, I’ll hopefully have some bug fixes out tonight or tomorrow night
